.eslintrc.json

```diff
@@ -14,27 +14,17 @@
     },
     "sourceType": "module"
   },
-  "plugins": [
-    "react"
-  ],
+  "plugins": ["react"],
   "rules": {
     "react/jsx-uses-react": "error",
     "react/jsx-uses-vars": "error",
-    "indent": [
-      "error",
-      2
-    ],
-    "linebreak-style": [
-      "error",
-      "unix"
-    ],
-    "quotes": [
-      "error",
-      "single"
-    ]
+    "indent": "off",
```

Comment:
I turned this off, as it interfered with Prettier.

I'm a huge fan of ESLint, and usually enable a lot of plugins, but formatting should be left for Prettier.